static/js/unread_ops.js

```diff
@@ -73,6 +74,53 @@ export function process_read_messages_event(message_ids) {
     unread_ui.update_unread_counts();
 }
 
+export function process_unread_messages_event({message_ids, message_details}) {
+    // This is the reverse of  process_unread_messages_event.
+    message_ids = unread.get_read_message_ids(message_ids);
+    if (message_ids.length === 0) {
+        return;
+    }
+
+    for (const message_id of message_ids) {
+        const message = message_store.get(message_id);
+
+        if (message) {
+            message.unread = true;
+        }
+
+        const message_info = message_details[message_id];
```

Comment:
Maybe `const event_details` would be a clearer variable name here?  I kinda want this to feel more different from the `message` object in naming.

Reply:
I think `event_details` would have essentially the same problem here, as it might mislead you to thinking it's **all** of the details of the event.  I think `message_details` makes it clear that we are talking about the `message_details` data structure that's in the event, in our zulip.yaml, in our chat debates, etc.

I definitely can relate to the name-soup concern here, but I don't think we should change this.